tutorials/cp-cap-java-service-reuse/cp-cap-java-service-reuse.md

```diff
@@ -106,35 +106,32 @@ Now that you have created your bookstore project, you need to define the domain
 
     ```CDS
     namespace sap.capire.bookstore;
-
+    
```

Comment:
I'd suppose the remove those blank lines. I understand that it improves readability, but in a tutorial where this is copied, it just increases the size of the document.